aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfiguration.java

```diff
@@ -33,19 +30,16 @@
 /**
  * Configuration options for {@link AWSDataStorePlugin}.
  */
+@SuppressWarnings("WeakerAccess") // This is a public API. Public methods available to user's package(s).
```

Comment:
One day, I'll know when this suppression is necessary and when it isn't. ;) Still on my list to dig into https://github.com/aws-amplify/amplify-android/pull/442/files#r422430836.

Reply:
Oops, I had missed those. Added some info here:

https://github.com/aws-amplify/amplify-android/pull/442/files#r426205557